NewsApp/app/src/main/AndroidManifest.xml

```diff
@@ -5,16 +5,19 @@
     <uses-permission android:name="android.permission.INTERNET"/>
 
     <application
+        android:name=".NewsApp"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:name=".NewsApp"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.NewsApp">
+        android:theme="@style/Theme.NewsApp" >
+        <activity
+            android:name=".features.news.NewsActivity"
+            android:exported="false" />
         <activity
-            android:name=".MainActivity"
-            android:exported="true">
+            android:name=".features.auth.AuthActivity"
+            android:exported="true" >
```

Comment:
should this be false?

Reply:
When I made it false it gave me a compilation error and says:
"A launchable activity must be exported as of Android 12, which also makes it available to other apps."